src/Command/RecipesCommand.php

```diff
@@ -30,10 +30,10 @@ class RecipesCommand extends BaseCommand
     /** @var \Symfony\Flex\Flex */
     private $flex;
 
-    private $symfonyLock;
-    private $downloader;
+    private Lock $symfonyLock;
```

Comment:
No property constructor promotion?

Reply:
No intention to do this in this PR: I prefer keeping the diff focused.
Could come later on as part of a code modernization PR.

Reply:
Fine for me 👌🏻